pkg/standalone/runconfig/run_config_parser.go

```diff
@@ -0,0 +1,184 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package runconfig
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"sync"
+
+	"github.com/dapr/cli/utils"
+
+	"gopkg.in/yaml.v2"
+)
+
+// constants for the keys from the yaml file.
+const APPS = "apps"
+
+func (a *AppsRunConfig) ParseAppsConfig(configFile string) ([]map[string]string, error) {
+	bytes, err := os.ReadFile(configFile)
+	if err != nil {
+		return nil, err
+	}
+	err = yaml.Unmarshal(bytes, &a)
+	if err != nil {
+		return nil, err
+	}
+	keyMappings, err := a.getKeyMappingFromYaml(bytes)
+	if err != nil {
+		return nil, err
+	}
+	return keyMappings, nil
+}
+
+func (a *AppsRunConfig) ValidateRunConfig() error {
+	if a.Version == 0 {
+		return fmt.Errorf("required filed %q not found in the provided app config file", "version")
+	}
+	// validate all paths in commons.
+	err := utils.ValidateFilePaths(a.Common.ConfigFile, a.Common.ResourcesPath)
+	if err != nil {
+		return err
```

Comment:
wrap error?

Reply:
wrapped in actual  method `ValidateFilePaths `